### DISCUSSION TITLE
Projects (beta) - GroupBy Multiple Fields

### DISCUSSION THREAD

**Discussion by mitchelsellers** [community/community#6422] [Projects and Issues]
In looking at the best way to organize content, the list view is fantastic in most regards, but I find that I could really benefit from "double grouping".

The use case for this is a project whereby I have a "Client" named for each item, it would be great to be able to group by "Client" THEN within each client group by status.

The reasoning behind this is that I could better communicate quickly to the clients, without having to make a "board per client" and get a true at-a-glance view.

The expected tree view might look like this.

* Client A
  * Open Issues
    * Issue ___
    * Issue ___
   * Blocked Issues
     * Issue ____
 * Client B
   * Open Issues
     * Issue ___

This would be a huge win, although I know how complicated it might be.

**Comment by RonnyB71** [comment]
Agree. Another use case for this is to group by Milestone\Priority. Having the possibility for "double grouping" would solve many use cases like this.
